src/features/search/components/SearchResults.tsx

```diff
@@ -98,6 +100,8 @@ export const SearchResults: React.FC = () => {
     params?.offerTypes?.isEvent ||
     params?.offerTypes?.isThing
 
+  const hasDatesHours = params?.date || params?.timeRange
```

Comment:
Ce n'est pas un boolean. Il est même infer en `any`. Tu peux corriger ça facilement
```suggestion
  const hasDatesHours = Boolean(params?.date || params?.timeRange)
```

Reply:
Ah yes je l'aurai pas cru nécessaire ici mais effectivement